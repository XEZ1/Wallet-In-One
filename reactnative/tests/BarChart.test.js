// PieChart.test.js
import React from 'react';
import { render, fireEvent } from '@testing-library/react-native';
import BarChartWallet from '../screens/charts/BarChart';
import renderer from 'react-test-renderer';

describe('<BarChartWallet />', () => {
    it('snapshot test', () => {
        const snapshot = renderer.create(<BarChartWallet />).toJSON();
        expect(snapshot).toMatchSnapshot();
      });

      it('displays the text "Wallet-In-One"', () => {
        const { getByText } = render(<BarChartWallet />);
        const walletInOne = getByText('Wallet-In-One');
    
        expect(walletInOne).toBeDefined();
        expect(walletInOne.props.children).toEqual('Wallet-In-One');
      });

      it('switch chart button test', () => {
        const navigate = jest.fn();
        const { getByText } = render(<BarChartWallet navigation={{ navigate }} />);
        fireEvent.press(getByText('Switch Chart'));
        expect(navigate).toHaveBeenCalledWith('Pie Chart');
      })
  });
