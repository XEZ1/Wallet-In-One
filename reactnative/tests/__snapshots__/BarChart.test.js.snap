// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BarChartWallet /> snapshot test 1`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flexGrow": 1,
      "justifyContent": "center",
      "paddingBottom": 20,
    }
  }
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View>
    <Text
      style={
        Array [
          Object {
            "alignSelf": "center",
            "fontSize": 50,
            "fontWeight": "900",
            "paddingVertical": 10,
          },
          Object {
            "color": "black",
          },
        ]
      }
    >
      Wallet-In-One
    </Text>
    <Text
      style={
        Array [
          Object {
            "fontSize": 25,
            "fontWeight": "900",
            "textAlign": "left",
          },
          Object {
            "color": "black",
          },
        ]
      }
    >
      Amount: Â£
      345
    </Text>
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
    >
      <View
        accessibilityRole="button"
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {},
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#007AFF",
                  "fontSize": 18,
                  "margin": 8,
                  "textAlign": "center",
                },
              ]
            }
          >
            Switch Chart
          </Text>
        </View>
      </View>
    </View>
    <View
      className="VictoryContainer"
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      pointerEvents="box-none"
      style={
        Object {
          "position": "relative",
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="350"
        bbWidth="750"
        focusable={false}
        height={350}
        meetOrSlice={0}
        minX={0}
        minY={0}
        role="img"
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            Object {
              "height": "100%",
              "width": "100%",
            },
            Object {
              "flex": 0,
              "height": 350,
              "width": 750,
            },
          ]
        }
        vbHeight={350}
        vbWidth={750}
        width={750}
      >
        <RNSVGGroup>
          <RNSVGRect
            fill={null}
            height="350"
            propList={
              Array [
                "fill",
              ]
            }
            width="750"
            x="0"
            y="0"
          />
          <RNSVGGroup>
            <RNSVGLine
              fill={
                Object {
                  "payload": 0,
                  "type": 0,
                }
              }
              propList={
                Array [
                  "fill",
                  "stroke",
                  "strokeWidth",
                  "strokeLinecap",
                  "strokeLinejoin",
                ]
              }
              stroke={
                Object {
                  "payload": 4294901760,
                  "type": 0,
                }
              }
              strokeLinecap={1}
              strokeLinejoin={1}
              strokeWidth="2"
              vectorEffect={1}
              x1="50"
              x2="50"
              y1="50"
              y2="300"
            />
            <RNSVGGroup>
              <RNSVGLine
                fill={null}
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeDasharray",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4294901760,
                    "type": 0,
                  }
                }
                strokeDasharray={
                  Array [
                    "10",
                    "5",
                  ]
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                vectorEffect={1}
                x1="50"
                x2="700"
                y1="237.5"
                y2="237.5"
              />
              <RNSVGLine
                fill={
                  Object {
                    "payload": 0,
                    "type": 0,
                  }
                }
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeWidth",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4287669422,
                    "type": 0,
                  }
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                strokeWidth="1"
                vectorEffect={1}
                x1="50"
                x2="45"
                y1="237.5"
                y2="237.5"
              />
              <RNSVGText
                content={null}
                dx={
                  Array [
                    "0",
                  ]
                }
                dy={Array []}
                font={Object {}}
                name="chart-axis-0-tickLabels-0"
                rotate={Array []}
                x={
                  Array [
                    "37",
                  ]
                }
                y={
                  Array [
                    "241.76",
                  ]
                }
              >
                <RNSVGTSpan
                  content="50"
                  dx={
                    Array [
                      "0",
                    ]
                  }
                  dy={
                    Array [
                      "0",
                    ]
                  }
                  fill={
                    Object {
                      "payload": 4278190080,
                      "type": 0,
                    }
                  }
                  font={
                    Object {
                      "fontFamily": "Helvetica Neue",
                      "fontSize": "12",
                      "letterSpacing": "normal",
                      "textAnchor": "end",
                    }
                  }
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotate={Array []}
                  x={
                    Array [
                      "37",
                    ]
                  }
                  y={Array []}
                />
              </RNSVGText>
            </RNSVGGroup>
            <RNSVGGroup>
              <RNSVGLine
                fill={null}
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeDasharray",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4294901760,
                    "type": 0,
                  }
                }
                strokeDasharray={
                  Array [
                    "10",
                    "5",
                  ]
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                vectorEffect={1}
                x1="50"
                x2="700"
                y1="175"
                y2="175"
              />
              <RNSVGLine
                fill={
                  Object {
                    "payload": 0,
                    "type": 0,
                  }
                }
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeWidth",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4287669422,
                    "type": 0,
                  }
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                strokeWidth="1"
                vectorEffect={1}
                x1="50"
                x2="45"
                y1="175"
                y2="175"
              />
              <RNSVGText
                content={null}
                dx={
                  Array [
                    "0",
                  ]
                }
                dy={Array []}
                font={Object {}}
                name="chart-axis-0-tickLabels-1"
                rotate={Array []}
                x={
                  Array [
                    "37",
                  ]
                }
                y={
                  Array [
                    "179.26",
                  ]
                }
              >
                <RNSVGTSpan
                  content="100"
                  dx={
                    Array [
                      "0",
                    ]
                  }
                  dy={
                    Array [
                      "0",
                    ]
                  }
                  fill={
                    Object {
                      "payload": 4278190080,
                      "type": 0,
                    }
                  }
                  font={
                    Object {
                      "fontFamily": "Helvetica Neue",
                      "fontSize": "12",
                      "letterSpacing": "normal",
                      "textAnchor": "end",
                    }
                  }
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotate={Array []}
                  x={
                    Array [
                      "37",
                    ]
                  }
                  y={Array []}
                />
              </RNSVGText>
            </RNSVGGroup>
            <RNSVGGroup>
              <RNSVGLine
                fill={null}
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeDasharray",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4294901760,
                    "type": 0,
                  }
                }
                strokeDasharray={
                  Array [
                    "10",
                    "5",
                  ]
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                vectorEffect={1}
                x1="50"
                x2="700"
                y1="112.5"
                y2="112.5"
              />
              <RNSVGLine
                fill={
                  Object {
                    "payload": 0,
                    "type": 0,
                  }
                }
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeWidth",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4287669422,
                    "type": 0,
                  }
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                strokeWidth="1"
                vectorEffect={1}
                x1="50"
                x2="45"
                y1="112.5"
                y2="112.5"
              />
              <RNSVGText
                content={null}
                dx={
                  Array [
                    "0",
                  ]
                }
                dy={Array []}
                font={Object {}}
                name="chart-axis-0-tickLabels-2"
                rotate={Array []}
                x={
                  Array [
                    "37",
                  ]
                }
                y={
                  Array [
                    "116.76",
                  ]
                }
              >
                <RNSVGTSpan
                  content="150"
                  dx={
                    Array [
                      "0",
                    ]
                  }
                  dy={
                    Array [
                      "0",
                    ]
                  }
                  fill={
                    Object {
                      "payload": 4278190080,
                      "type": 0,
                    }
                  }
                  font={
                    Object {
                      "fontFamily": "Helvetica Neue",
                      "fontSize": "12",
                      "letterSpacing": "normal",
                      "textAnchor": "end",
                    }
                  }
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotate={Array []}
                  x={
                    Array [
                      "37",
                    ]
                  }
                  y={Array []}
                />
              </RNSVGText>
            </RNSVGGroup>
            <RNSVGGroup>
              <RNSVGLine
                fill={null}
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeDasharray",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4294901760,
                    "type": 0,
                  }
                }
                strokeDasharray={
                  Array [
                    "10",
                    "5",
                  ]
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                vectorEffect={1}
                x1="50"
                x2="700"
                y1="50"
                y2="50"
              />
              <RNSVGLine
                fill={
                  Object {
                    "payload": 0,
                    "type": 0,
                  }
                }
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeWidth",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4287669422,
                    "type": 0,
                  }
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                strokeWidth="1"
                vectorEffect={1}
                x1="50"
                x2="45"
                y1="50"
                y2="50"
              />
              <RNSVGText
                content={null}
                dx={
                  Array [
                    "0",
                  ]
                }
                dy={Array []}
                font={Object {}}
                name="chart-axis-0-tickLabels-3"
                rotate={Array []}
                x={
                  Array [
                    "37",
                  ]
                }
                y={
                  Array [
                    "54.26",
                  ]
                }
              >
                <RNSVGTSpan
                  content="200"
                  dx={
                    Array [
                      "0",
                    ]
                  }
                  dy={
                    Array [
                      "0",
                    ]
                  }
                  fill={
                    Object {
                      "payload": 4278190080,
                      "type": 0,
                    }
                  }
                  font={
                    Object {
                      "fontFamily": "Helvetica Neue",
                      "fontSize": "12",
                      "letterSpacing": "normal",
                      "textAnchor": "end",
                    }
                  }
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotate={Array []}
                  x={
                    Array [
                      "37",
                    ]
                  }
                  y={Array []}
                />
              </RNSVGText>
            </RNSVGGroup>
          </RNSVGGroup>
          <RNSVGGroup>
            <RNSVGLine
              fill={
                Object {
                  "payload": 0,
                  "type": 0,
                }
              }
              propList={
                Array [
                  "fill",
                  "stroke",
                  "strokeWidth",
                  "strokeLinecap",
                  "strokeLinejoin",
                ]
              }
              stroke={
                Object {
                  "payload": 4294901760,
                  "type": 0,
                }
              }
              strokeLinecap={1}
              strokeLinejoin={1}
              strokeWidth="2"
              vectorEffect={1}
              x1="50"
              x2="700"
              y1="300"
              y2="300"
            />
            <RNSVGGroup>
              <RNSVGLine
                fill={null}
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeDasharray",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4294901760,
                    "type": 0,
                  }
                }
                strokeDasharray={
                  Array [
                    "10",
                    "5",
                  ]
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                vectorEffect={1}
                x1="50"
                x2="50"
                y1="300"
                y2="50"
              />
              <RNSVGLine
                fill={
                  Object {
                    "payload": 0,
                    "type": 0,
                  }
                }
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeWidth",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4287669422,
                    "type": 0,
                  }
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                strokeWidth="1"
                vectorEffect={1}
                x1="50"
                x2="50"
                y1="300"
                y2="305"
              />
              <RNSVGText
                content={null}
                dx={
                  Array [
                    "0",
                  ]
                }
                dy={Array []}
                font={Object {}}
                name="chart-axis-1-tickLabels-0"
                rotate={Array []}
                x={
                  Array [
                    "50",
                  ]
                }
                y={
                  Array [
                    "323.26",
                  ]
                }
              >
                <RNSVGTSpan
                  content="Credit Card"
                  dx={
                    Array [
                      "0",
                    ]
                  }
                  dy={
                    Array [
                      "0",
                    ]
                  }
                  fill={
                    Object {
                      "payload": 4278190080,
                      "type": 0,
                    }
                  }
                  font={
                    Object {
                      "fontFamily": "Helvetica Neue",
                      "fontSize": "12",
                      "letterSpacing": "normal",
                      "textAnchor": "middle",
                    }
                  }
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotate={Array []}
                  x={
                    Array [
                      "50",
                    ]
                  }
                  y={Array []}
                />
              </RNSVGText>
            </RNSVGGroup>
            <RNSVGGroup>
              <RNSVGLine
                fill={null}
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeDasharray",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4294901760,
                    "type": 0,
                  }
                }
                strokeDasharray={
                  Array [
                    "10",
                    "5",
                  ]
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                vectorEffect={1}
                x1="266.66666666666663"
                x2="266.66666666666663"
                y1="300"
                y2="50"
              />
              <RNSVGLine
                fill={
                  Object {
                    "payload": 0,
                    "type": 0,
                  }
                }
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeWidth",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4287669422,
                    "type": 0,
                  }
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                strokeWidth="1"
                vectorEffect={1}
                x1="266.66666666666663"
                x2="266.66666666666663"
                y1="300"
                y2="305"
              />
              <RNSVGText
                content={null}
                dx={
                  Array [
                    "0",
                  ]
                }
                dy={Array []}
                font={Object {}}
                name="chart-axis-1-tickLabels-1"
                rotate={Array []}
                x={
                  Array [
                    "266.66666666666663",
                  ]
                }
                y={
                  Array [
                    "323.26",
                  ]
                }
              >
                <RNSVGTSpan
                  content="Crypto"
                  dx={
                    Array [
                      "0",
                    ]
                  }
                  dy={
                    Array [
                      "0",
                    ]
                  }
                  fill={
                    Object {
                      "payload": 4278190080,
                      "type": 0,
                    }
                  }
                  font={
                    Object {
                      "fontFamily": "Helvetica Neue",
                      "fontSize": "12",
                      "letterSpacing": "normal",
                      "textAnchor": "middle",
                    }
                  }
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotate={Array []}
                  x={
                    Array [
                      "266.66666666666663",
                    ]
                  }
                  y={Array []}
                />
              </RNSVGText>
            </RNSVGGroup>
            <RNSVGGroup>
              <RNSVGLine
                fill={null}
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeDasharray",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4294901760,
                    "type": 0,
                  }
                }
                strokeDasharray={
                  Array [
                    "10",
                    "5",
                  ]
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                vectorEffect={1}
                x1="483.3333333333333"
                x2="483.3333333333333"
                y1="300"
                y2="50"
              />
              <RNSVGLine
                fill={
                  Object {
                    "payload": 0,
                    "type": 0,
                  }
                }
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeWidth",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4287669422,
                    "type": 0,
                  }
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                strokeWidth="1"
                vectorEffect={1}
                x1="483.3333333333333"
                x2="483.3333333333333"
                y1="300"
                y2="305"
              />
              <RNSVGText
                content={null}
                dx={
                  Array [
                    "0",
                  ]
                }
                dy={Array []}
                font={Object {}}
                name="chart-axis-1-tickLabels-2"
                rotate={Array []}
                x={
                  Array [
                    "483.3333333333333",
                  ]
                }
                y={
                  Array [
                    "323.26",
                  ]
                }
              >
                <RNSVGTSpan
                  content="Stocks"
                  dx={
                    Array [
                      "0",
                    ]
                  }
                  dy={
                    Array [
                      "0",
                    ]
                  }
                  fill={
                    Object {
                      "payload": 4278190080,
                      "type": 0,
                    }
                  }
                  font={
                    Object {
                      "fontFamily": "Helvetica Neue",
                      "fontSize": "12",
                      "letterSpacing": "normal",
                      "textAnchor": "middle",
                    }
                  }
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotate={Array []}
                  x={
                    Array [
                      "483.3333333333333",
                    ]
                  }
                  y={Array []}
                />
              </RNSVGText>
            </RNSVGGroup>
            <RNSVGGroup>
              <RNSVGLine
                fill={null}
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeDasharray",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4294901760,
                    "type": 0,
                  }
                }
                strokeDasharray={
                  Array [
                    "10",
                    "5",
                  ]
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                vectorEffect={1}
                x1="700"
                x2="700"
                y1="300"
                y2="50"
              />
              <RNSVGLine
                fill={
                  Object {
                    "payload": 0,
                    "type": 0,
                  }
                }
                propList={
                  Array [
                    "fill",
                    "stroke",
                    "strokeWidth",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  Object {
                    "payload": 4287669422,
                    "type": 0,
                  }
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                strokeWidth="1"
                vectorEffect={1}
                x1="700"
                x2="700"
                y1="300"
                y2="305"
              />
              <RNSVGText
                content={null}
                dx={
                  Array [
                    "0",
                  ]
                }
                dy={Array []}
                font={Object {}}
                name="chart-axis-1-tickLabels-3"
                rotate={Array []}
                x={
                  Array [
                    "700",
                  ]
                }
                y={
                  Array [
                    "323.26",
                  ]
                }
              >
                <RNSVGTSpan
                  content="Debit Card"
                  dx={
                    Array [
                      "0",
                    ]
                  }
                  dy={
                    Array [
                      "0",
                    ]
                  }
                  fill={
                    Object {
                      "payload": 4278190080,
                      "type": 0,
                    }
                  }
                  font={
                    Object {
                      "fontFamily": "Helvetica Neue",
                      "fontSize": "12",
                      "letterSpacing": "normal",
                      "textAnchor": "middle",
                    }
                  }
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotate={Array []}
                  x={
                    Array [
                      "700",
                    ]
                  }
                  y={Array []}
                />
              </RNSVGText>
            </RNSVGGroup>
          </RNSVGGroup>
          <RNSVGGroup>
            <RNSVGPath
              d="M 22.916666666666668, 300 
A 0 0 0 0 1, 22.916666666666668, 300 
L 22.916666666666668, 50 
A 0 0 0 0 1, 22.916666666666668, 50 
L 77.08333333333333, 50 
A 0 0 0 0 1, 77.08333333333333, 50 
L 77.08333333333333, 300 
A 0 0 0 0 1, 77.08333333333333, 300 
 z"
              fill={
                Object {
                  "payload": 4278190335,
                  "type": 0,
                }
              }
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              propList={
                Array [
                  "fill",
                  "stroke",
                  "strokeWidth",
                ]
              }
              responsible={true}
              stroke={
                Object {
                  "payload": 4278190335,
                  "type": 0,
                }
              }
              strokeWidth="0"
            />
            <RNSVGPath
              d="M 239.5833333333333, 300 
A 0 0 0 0 1, 239.5833333333333, 300 
L 239.5833333333333, 237.5 
A 0 0 0 0 1, 239.5833333333333, 237.5 
L 293.74999999999994, 237.5 
A 0 0 0 0 1, 293.74999999999994, 237.5 
L 293.74999999999994, 300 
A 0 0 0 0 1, 293.74999999999994, 300 
 z"
              fill={
                Object {
                  "payload": 4278190335,
                  "type": 0,
                }
              }
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              propList={
                Array [
                  "fill",
                  "stroke",
                  "strokeWidth",
                ]
              }
              responsible={true}
              stroke={
                Object {
                  "payload": 4278190335,
                  "type": 0,
                }
              }
              strokeWidth="0"
            />
            <RNSVGPath
              d="M 456.25, 300 
A 0 0 0 0 1, 456.25, 300 
L 456.25, 206.25 
A 0 0 0 0 1, 456.25, 206.25 
L 510.41666666666663, 206.25 
A 0 0 0 0 1, 510.41666666666663, 206.25 
L 510.41666666666663, 300 
A 0 0 0 0 1, 510.41666666666663, 300 
 z"
              fill={
                Object {
                  "payload": 4278190335,
                  "type": 0,
                }
              }
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              propList={
                Array [
                  "fill",
                  "stroke",
                  "strokeWidth",
                ]
              }
              responsible={true}
              stroke={
                Object {
                  "payload": 4278190335,
                  "type": 0,
                }
              }
              strokeWidth="0"
            />
            <RNSVGPath
              d="M 672.9166666666666, 300 
A 0 0 0 0 1, 672.9166666666666, 300 
L 672.9166666666666, 275 
A 0 0 0 0 1, 672.9166666666666, 275 
L 727.0833333333334, 275 
A 0 0 0 0 1, 727.0833333333334, 275 
L 727.0833333333334, 300 
A 0 0 0 0 1, 727.0833333333334, 300 
 z"
              fill={
                Object {
                  "payload": 4278190335,
                  "type": 0,
                }
              }
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              propList={
                Array [
                  "fill",
                  "stroke",
                  "strokeWidth",
                ]
              }
              responsible={true}
              stroke={
                Object {
                  "payload": 4278190335,
                  "type": 0,
                }
              }
              strokeWidth="0"
            />
          </RNSVGGroup>
          <View
            pointerEvents="box-none"
            style={
              Object {
                "left": 0,
                "position": "absolute",
                "top": 0,
                "zIndex": 99,
              }
            }
          >
            <RNSVGSvgView
              align="xMidYMid"
              bbHeight="350"
              bbWidth="750"
              focusable={false}
              height={350}
              meetOrSlice={0}
              minX={0}
              minY={0}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                    "borderWidth": 0,
                  },
                  Object {
                    "height": "100%",
                    "overflow": "visible",
                    "width": "100%",
                  },
                  Object {
                    "flex": 0,
                    "height": 350,
                    "width": 750,
                  },
                ]
              }
              vbHeight={350}
              vbWidth={750}
              width={750}
            >
              <RNSVGGroup />
            </RNSVGSvgView>
          </View>
        </RNSVGGroup>
      </RNSVGSvgView>
    </View>
  </View>
</RCTScrollView>
`;
